### ISSUE TITLE
exec breaks highlighting

### ISSUE THREAD

**Issue by bomgar** [ObserverOfTime/tree-sitter-ssh-config#6]
```


Host demo.demo.de demo
    HostName 10.187.5.43

Match host 10.177.18.*,11.187.1.* exec "$SHELL ~/.ssh/demo.sh %h"

Host *.bla.de | *.bla-dev.de
    User meep
    HostKeyAlgorithms +ssh-rsa
    PubkeyAcceptedKeyTypes +ssh-rsa
    StrictHostKeyChecking no
```
![2023-11-02-171858_grim](https://github.com/ObserverOfTime/tree-sitter-ssh-config/assets/402052/a7519e9e-edb4-4d5f-97cf-f526a7700350)

**Comment by ObserverOfTime** [comment]
Empty `Match` or `Host` blocks are not allowed.

**Comment by bomgar** [comment]
What do you mean they are not allowed? This works perfectly fine.

**Comment by ObserverOfTime** [comment]
All right, fixed in 9dcfe0f.

**Comment by bomgar** [comment]
Thank you
